proteobench/io/parsing/parse_ion.py

```diff
@@ -420,10 +420,7 @@ def _load_prolinestudio_msangel(input_csv: str) -> pd.DataFrame:
         The loaded dataframe.
     """
     input_data_frame = pd.read_excel(
-        input_csv,
-        sheet_name="Quantified peptide ions",
-        header=0,
-        index_col=None,
+        input_csv, sheet_name="Quantified peptide ions", header=0, index_col=None, engine="calamine"
```

Comment:
if you put a comma to the end, `black` will format it again on several lines making the diff a bit clearer.